documentation/src/main/asciidoc/topics/ref_cache_store_sfs.adoc

```diff
@@ -1,14 +1,17 @@
 [id='sfs_cache_store']
 = Single File Cache Stores
-Single File cache stores, `SingleFileStore`, persist data to a single file and maintains an in-memory index of keys and their values in that file.
+Single File cache stores, `SingleFileStore`, persist data to file. {brandname}
+also maintains an in-memory index of keys and their values in the store. By
+default, Single File cache stores are segmented, which means that {brandname}
```

Comment:
The second sentence is not that clear imo. The store maintains an in-memory index of keys, whilst key/values are stored in the file.